File: Curso_logica_de_programacao/Exercícios_logica_em_Python/Soma_vetores/Soma_vetores.py
print("Quantos numeros voce vai digitar? ", end="")
n = int(input())

x = [0 for x in range(n)]

for i in range(0, n):
    x[i] = float(input("Digite um numero: "))


soma = 0
for i in range(0, n):
    soma += x[i]


print()
print("Valores = ", end="")
for i in range(0, n):
    print(x[i], end=" ")

print()
media: float
media = soma / n
print(f"soma = {soma}")
print(f"Media = {media}")

File: Algoritmos_e_Logica_de_Programacao/Exercicios_logica_em_C/04-vetores/soma_vetores/soma_vetores.c
// App para somar numeros de vetores diferentes, recebe os numeros para o vetor A e depois para o vetor B e em sequencia soma os numeros dos vetores (A+B)

#include <stdio.h>

int main()
{

    int total_numeros;

    printf("Quantos valores voce vai digitar em cada vetor? ");
    scanf("%d", &total_numeros);

    printf("Digite os valores do vetor A: \n");
    int vetorA[total_numeros];
    for (int i = 0; i < total_numeros; i++) {
        scanf("%d", &vetorA[i]);
    }

    printf("Digite os valores do vetor B: \n");
    int vetorB[total_numeros];
    for (int i = 0; i < total_numeros; i++) {
        scanf("%d", &vetorB[i]);
    }

    int resultado[total_numeros];
    for (int i = 0; i < total_numeros; i++) {
        resultado[i] = vetorA[i] + vetorB[i];
    }

     printf("Vetor resultante: ");
    for (int i = 0; i < total_numeros; i++) {
        printf("%d\n", resultado[i]);
    }

    return 0;
}